pkg/cloud/aws/bootstrap/pod.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: aws-cloud-controller-manager
+  namespace: kube-system
+spec:
+  priorityClassName: system-cluster-critical
+  containers:
+  - args:
+    - --cloud-provider=aws
+    - --use-service-account-credentials=false
+    - --controllers=*,-service,-route # disable service and route controllers in bootstrap
+    - --kubeconfig=/etc/kubernetes/secrets/kubeconfig
+    - --leader-elect=false
+    - -v=2
+    - "2> /var/log/bootstrap-control-plane/cloud-controller-manager.log"
+    image: gcr.io/k8s-staging-provider-aws/cloud-controller-manager:v1.19.0-alpha.1
```

Comment:
Guess we should use ocp image there. Don't see substitution call in aws.GetBootstrapResources...

Reply:
It would be there, if we add pod substitution pattern. We currently substitute only Deployments. The resource goes through substitution cycle in https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/45/files#diff-0f4e2af2ebf3a82b1841788f7aad091a1e46fdee85a28a07c9747eb514dbd83fR68